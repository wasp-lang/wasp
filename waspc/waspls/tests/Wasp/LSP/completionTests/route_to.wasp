//! test/completion
app todoApp {
  wasp: {
    version: "^0.10.5", 
  },
  title: "todo!",
}

route MainRoute { path: "/", to: | }
                                 ^
page MainPage {
  component: import { MainPage } from "@client/MainPage.jsx",
}
