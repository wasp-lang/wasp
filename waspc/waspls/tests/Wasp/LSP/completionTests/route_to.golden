Completion items:
  label={"MainPage"} kind={CiVariable} detail={":: page (declaration type)"}
  label={"MainRoute"} kind={CiVariable} detail={":: route (declaration type)"}
  label={"todoApp"} kind={CiVariable} detail={":: app (declaration type)"}
